src/components/elements/__tests__/Link.test.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { StyleSheetTestUtils } from 'aphrodite'
+import { render } from 'enzyme'
+import { TestRouter } from '../../../__mocks__'
+import { Link } from '..'
```

Comment:
for tests, we should avoid importing and evaluating extra code, so import statements in all tests should be made this way: ```import Link from '../Link'```